### PR TITLE
Added Sergeant Pepper to the list

### PR DESCRIPTION
**Pull request by vybhav72954**

### CODE REVIEW

File: src/recipes/sergeant-pepper.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+    "name": "<Sergeant Pepper>",
```
Comment (alfg) [inline src/recipes/sergeant-pepper.json:2]:
```suggestion
    "name": "Sergeant Pepper",
```

You can remove these symbols for all fields as these were just placeholders as an example. Thanks.

File: src/recipes/sergeant-pepper.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        "The Ingredients are for 2 servings of Sergeant Pepper",
+        "On a Saucepan, toast cumin seeds until they just begin to darken.",
+        "Add water, sugar, and black pepper, regularly stir to dissolve sugar.",
+        "Simmer for 15 minutes, strain, and let cool, This is the Cumin-Pepper Syrup",
+        "Add Pineapple Juice, Orgeat, Lime Juice, Cumin-Pepper Syrup, Gin and some extra pepper according to taste",
+        "Add Ic Cubes, Shake well",
+        "Strain and Serve immediately"
+    ],
+    "image": "<sergeant-pepper.jpg>",
+    "source": "<source url if available, otherwise remove this line>",
```
Comment (alfg) [inline src/recipes/sergeant-pepper.json:67]: Remove this line if you don't have a source.

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, left a couple of suggestions. Thanks.

**Comment by vybhav72954** [comment]
Done, KIndly see if the final commit suits the requirement. @alfg

File: src/recipes/sergeant-pepper.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    "directions": [
+        "The Ingredients are for 2 servings of Sergeant Pepper",
+        "On a Saucepan, toast cumin seeds until they just begin to darken.",
+        "Add water, sugar, and black pepper, regularly stir to dissolve sugar.",
+        "Simmer for 15 minutes, strain, and let cool, This is the Cumin-Pepper Syrup",
+        "Add Pineapple Juice, Orgeat, Lime Juice, Cumin-Pepper Syrup, Gin and some extra pepper according to taste",
+        "Add Ic Cubes, Shake well",
+        "Strain and Serve immediately"
+    ],
+    "image": "sergeant-pepper.jpg>",
```
Comment (alfg) [inline src/recipes/sergeant-pepper.json:66]:
```suggestion
    "image": "sergeant-pepper.jpg",
```

**Review by alfg — CHANGES_REQUESTED** [review]

**Comment by alfg** [comment]
Added another suggested change.

**Comment by alfg** [comment]
Thanks!

**Comment by vybhav72954** [comment]
Sorry for such childish mistakes. Thank You for your suggestions.

**Comment by alfg** [comment]
No worries! Thanks for the contribution! :)

**Pull request merged**